### PR TITLE
Create project level README.md

### PR DESCRIPTION
**Pull request by Rozzii**
This was missing from the project and folks who find us directly via GitHub would be welcomed by this README first.

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+Metal³ is one of the providers for the Kubernetes sub-project [Cluster
+API](https://github.com/kubernetes-sigs/cluster-api). Cluster API provides
+infrastructure agnostic Kubernetes lifecycle management, and Metal³ brings the
+bare metal implementation.
+
+Metal³ is open-source and welcomes community contributions.
+
+## Contacts
+
+* [Website](https://github.com/metal3-io)
```
Comment (tuminoid) [inline profile/README.md]: I would not say our website is organization page in Github, but https://metal3.io/ especially when this readme is being displayed in Github org page already :)

```suggestion
* [Website](https://metal3.io/)
```

**Comment by dtantsur** [comment]
Looks good except for the issue that Tuomo spotted.

File: profile/README.md (hunk 1 above)
Comment (Rozzii) [reply profile/README.md]: done thanks!

**Review by tuminoid — COMMENTED** [review]
/approve

**Pull request merged**